src/main/java/au/com/dmg/fusion/util/InstantAdapter.java

```diff
@@ -47,6 +53,7 @@ public Instant fromJson2(String string) {
 
     @ToJson
     public String toJson2(Instant value) {
-        return value.toString();
+//        return value.toString();
+        return formatter.format(value);
```

Comment:
Can you please place a sample formatted value as a comment - just so we can easily see a sample output?  

return formatter.format(value);  **//sample formatted value: _[replace with a valid value here]_**